index.js

```diff
@@ -131,6 +131,16 @@ function _readFC6(data, next) {
         next(null, {"address": dataAddress, "value": value});
 }
 
+function _readFC15(data, next) {
+    var dataAddress = data.readUInt16BE(2);
+    var value = data.readUInt16BE(4);
+
+    if(next)
+        next(null, {"address": dataAddress, "value": value});
```

Comment:
Yes, here the "value" should be "length". It was my carelessness, thanks you for the correction :-)